seahorn/jobs/array_list_shrink_to_fit/aws_array_list_shrink_to_fit_harness.c

```diff
@@ -14,10 +14,10 @@
 int main() {
     /* data structure */
     struct aws_array_list list;
-    initialize_array_list(&list);
+    initialize_bounded_array_list(&list, MAX_INITIAL_SIZE,
+        MAX_INITIAL_ITEM_ALLOCATION, MAX_ITEM_SIZE);
```

Comment:
I do not like these macros all over the place. This comment applies earlier. Push them into `initialize_bounded_array_list`. Rename the function into `mk_bnd_array_list(struct aws_array_list *list);`

Use the bounds from the library instead from the macros.

Reply:
Will change the parameter list of this function. Thanks for the suggestion.